lib/cubits/database_connection/database_connection_cubit.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:bloc/bloc.dart';
+import 'package:coffeecard/base/strings.dart';
+import 'package:coffeecard/data/repositories/v1/app_config_repository.dart';
+import 'package:coffeecard/generated/api/coffeecard_api_v2.swagger.swagger.dart';
+import 'package:coffeecard/service_locator.dart';
+import 'package:equatable/equatable.dart';
+import 'package:flutter/material.dart';
+
+part 'database_connection_state.dart';
+
+class DatabaseConnectionCubit extends Cubit<DatabaseConnectionState> {
```

Comment:
I'm not in favour of using the name `DatabaseConnection` in this PR. It does not capture the full impact of being connected to a test server API. Being connected to a `Test Environment` also means that all servers calls (when implemented) will go to the MobilePay Sandbox, no money will be withdrawn etc.

I therefore suggest rename `DatabaseConnection` to e.g.
- EnvironmentType
- ServerEnviornmentType
- ServerEnvironment
- Possibly there are also better suggestions than this

Reply:
renamed to EnvironmentType